### DISCUSSION TITLE
How to write string with Html tags in a cell?

### DISCUSSION THREAD

**Discussion by nikheelneelay** [exceljs/exceljs#2475] [Q&A]
## 💬 Questions and Help

Hi, I am trying to insert string with html tags in it and expect it to render it as html.
Input : 
![image](https://user-images.githubusercontent.com/16623762/81372056-fc836900-9116-11ea-815d-00cf28b954ba.png)

Current Output: 
![image](https://user-images.githubusercontent.com/16623762/81371791-6a7b6080-9116-11ea-8ad2-a22e58d598a2.png)
Expected Output:
![image](https://user-images.githubusercontent.com/16623762/81371835-82eb7b00-9116-11ea-8eac-a04bed914b27.png)
My question is, if i provide an html markup as input, can it be be show as html in excel using exceljs in stead just showing markup tags?

**Comment by rdzakmic** [comment]
I would like to know as well is there possibility for this to work?

**Comment by Yabonev** [comment]
Hello, I have the same use case. Using Excel directly, there's _Paste Special_, from which I can choose _HTML_, which **"Inserts the contents of the Clipboard as HTML format"**. Is there a way to configure a cell to use **HTML format** using exceljs?

**Comment by subhaneetshrestha** [comment]
> Hello, I have the same use case. Using Excel directly, there's _Paste Special_, from which I can choose _HTML_, which **"Inserts the contents of the Clipboard as HTML format"**. Is there a way to configure a cell to use **HTML format** using exceljs?

Did you find a way?

**Comment by veeresh745-sys** [comment]
Any work around or updates for this issue?

**Comment by tchopra91** [comment]
Any work around or updates for this issue?

**Comment by mohit000001** [comment]
Any update ?

**Comment by muftisami** [comment]
any update?

**Comment by Yabonev** [comment]
@subhaneetshrestha @veeresh745-sys @mohit000001  @muftisami  @tchopra91 My team and I couldn't find a way to write HTML to excel programmatically, we switched to a library called jsPDF and we're creating pdfs instead of excel. Sorry for the late, late, late reply. My suggestion is if you have a back-end to generate reports there using some other solution, it's been a real struggle for us to deal with report generation on the front-end.
